public/stylesheets/cards.css

```diff
@@ -128,3 +132,82 @@ fieldset {
 #continue-card-title {
   font-size: 25px;
 }
+
+@media (max-width: 1024px) {
+  .card-title {
+    font-size: 25px;
+  }
+  #gameGuesses > div {
+    margin-bottom: 8px;
+  }
+  .setting-fieldset > label {
+    margin-right: 15px;
+  }
+  #gameGuesses {
+    font-size: 13px;
+  }
+  #page-counter {
+    font-size: 20px;
+  }
+  .page-title {
+    font-size: 30px;
+  }
+  .card-button {
+    font-size: 19px;
+  }
+  .prev-session-info {
+    font-size: 16px;
+  }
+}
+
+@media (max-width: 576px) {
+  .page-title {
+    font-size: 19px;
+    padding: 5px;
+  }
+  .setting {
+    padding: 6px 15px;
+  }
+  .setting-fieldset {
+    font-size: 11px;
+  }
+  .setting-title {
+    font-size: 14px;
+  }
+  #page-counter {
```

Comment:
It's not centered and actually bothers me 
![image](https://github.com/DmitrijSidorkin/pixel-thing/assets/9388169/49084abd-79d1-4d62-92fa-70e67635f830)